python/taichi/lang/util.py

```diff
@@ -1,14 +1,13 @@
 from .core import taichi_lang_core
 import numpy as np
-import os
 
 _has_pytorch = False
-
-_env_torch = os.environ.get('TI_ENABLE_TORCH', '1')
+_env_torch = __import__('os').environ.get('TI_ENABLE_TORCH', '1')
```

Comment:
just curious: why this change here?

Reply:
Since we use the `from .util import *` pattern... we will get `ti.os` at the end.

Reply:
I see, so this is to resolve the potential monkey patching problem caused by `import *`? That makes sense to me. Again, I really hope we could get rid of `import *` patterns wherever possible : /

Reply:
> I see, so this is to resolve the potential monkey patching problem caused by `import *`? That makes sense to me. Again, I really hope we could get rid of `import *` patterns wherever possible : /

Agree, ``import *`` can be kinds of dangerous potentially